### PR TITLE
Muse Dash: Make which .net to download more explicit in setup guides.

### PR DESCRIPTION
**Pull request by DeamonHunter**
## What is this fixing or adding?
The .net 6.0 downloads page has a lot of downloads which can make it a bit difficult to determine which to download. The guide now explicitly references one of them.

## How was this tested?
Not Tested. Docs change.

Paging @ShinyNT to double check the spanish translation.

### CODE REVIEW

File: worlds/musedash/docs/setup_en.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
   - \[Optional\] [Just as Planned] DLC: [Also Available on Steam](https://store.steampowered.com/app/1055810/Muse_Dash__Just_as_planned/)
 - Melon Loader: [GitHub](https://github.com/LavaGang/MelonLoader/releases/latest)
   - .Net Framework 4.8 may be needed for the installer: [Download](https://dotnet.microsoft.com/en-us/download/dotnet-framework/net48)
-- .Net 6.0 (If not already installed): [Download](https://dotnet.microsoft.com/en-us/download/dotnet/6.0#runtime-6.0.15)
+- .NET Desktop Runtime 6.0.XX (If not already installed): [Download](https://dotnet.microsoft.com/en-us/download/dotnet/6.0#runtime-6.0.15)
```
Comment (Jarno458) [inline worlds/musedash/docs/setup_en.md:14]: Are you sure you want to hard pin this to version 6.0.15? i would expect it to just be latest version

Comment (DeamonHunter) [reply worlds/musedash/docs/setup_en.md]: Ah. I forgot about that, because it just ignores it anyway. Fixed.

**Comment by ShinyNT** [comment]
Spanish part is ok, thanks for letting me know. I'd maybe link the spanish page instead (https://dotnet.microsoft.com/es-es/download/dotnet/6.0) for convenience, but that's it.

**Comment by DeamonHunter** [comment]
Good call on the spanish links. Both links updated to the spanish pages. Also took the chance to update the [Just as Planned] links to the [Muse Plus] links now that [Just as Planned] is no longer buyable.

**Review by nicholassaylor — APPROVED** [review]
LGTM

**Review by ThePhar — APPROVED** [review]

**Pull request merged**